cli/src/templates/packages/tamagui/tamagui.config.ts.ejs

```diff
@@ -45,15 +45,14 @@ const bodyFont = createInterFont();
 		justifyContent: 'space-between',
 	});
 
-	export const Title = styled(Text, {
-		fontSize: 64,
-		fontWeight: 'bold',
-	});
+	export const Title = styled(H1, {
+    color: '#000',
+  });
 
-	export const Subtitle = styled(Text, {
-		color: '#38434D',
-		fontSize: 36,
-	});
+  export const Subtitle = styled(SizableText, {
+    color: '#38434D',
+    size: '$7',
```

Comment:
can you elaborate on how this $ notation works? I've never been able to figure it out.

Reply:
The `size` property is a custom property from Tamagui to manage all font properties according to the theme you defined.

It's use for this react properties:
* fontSize
* fontWeight
* lineHeight
* letterSpacing

In this case the token `$7` is define to apply this values: 
* fontSize: 20
* fontWeight: 600
* lineHeight: 30
* letterSpacing: 0

This values can be edited for each font you create for example:

```
{
    "family": "Inter, Helvetica, Arial, sans-serif",
    "lineHeight": {
        "1": 16,
        "2": 21,
        "3": 22,
        "4": 23,
        "5": 26,
        "6": 28,
        "7": 30,
        "8": 33,
        "9": 41,
        "10": 59,
        "11": 69,
        "12": 76,
        "13": 87,
        "14": 109,
        "15": 133,
        "16": 155,
        "true": 23
    },
    "weight": {
        "1": "400",
        "2": "400",
        "3": "400",
        "4": "400",
        "5": "400",
        "6": "400",
        "7": "600",
        "8": "600",
        "9": "600",
        "10": "600",
        "11": "600",
        "12": "600",
        "13": "600",
        "14": "600",
        "15": "600",
        "16": "600",
        "true": "600"
    },
    "letterSpacing": {
        "1": 0,
        "2": 0,
        "3": 0,
        "4": 0,
        "5": 0,
        "6": 0,
        "7": 0,
        "8": 0,
        "9": 0,
        "10": 0,
        "11": 0,
        "12": 0,
        "13": 0,
        "14": 0,
        "15": 0,
        "16": 0,
        "true": 0
    },
    "size": {
        "1": 11,
        "2": 12,
        "3": 13,
        "4": 14,
        "5": 16,
        "6": 18,
        "7": 20,
        "8": 23,
        "9": 30,
        "10": 46,
        "11": 55,
        "12": 62,
        "13": 72,
        "14": 92,
        "15": 114,
        "16": 134,
        "true": 14
    },
    "face": {
        "100": {
            "normal": "Inter, Helvetica, Arial, sans-serif"
        },
        "200": {
            "normal": "Inter, Helvetica, Arial, sans-serif"
        },
        "300": {
            "normal": "Inter Light",
            "italic": "Inter LightItalic"
        },
        "400": {
            "normal": "Inter",
            "italic": "Inter Italic"
        },
        "500": {
            "normal": "Inter",
            "italic": "Inter Italic"
        },
        "600": {
            "normal": "Inter",
            "italic": "Inter MediumItalic"
        },
        "700": {
            "normal": "Inter SemiBold",
            "italic": "Inter SemiBoldItalic"
        },
        "800": {
            "normal": "Inter Bold",
            "italic": "Inter BoldItalic"
        },
        "900": {
            "normal": "Inter Bold",
            "italic": "Inter BoldItalic"
        },
        "normal": {
            "normal": "Inter Regular",
            "italic": "Inter Italic"
        },
        "bold": {
            "normal": "Inter Bold",
            "italic": "Inter BoldItalic"
        }
    }
}
```

Reply:
thanks for the explanation! this makes sense :)